docs/devGuide/development/migratingNodeJs.md

```diff
@@ -0,0 +1,95 @@
+{% set title = "Migrating Node.js" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+Node.js versions have to upgraded periodically before the version being used reaches its [end of life](https://endoflife.date/nodejs).
+This page outlines the steps of migrating to a higher version of Node.js
+</div>
+
+### Preparation
+
+#### Choosing Node.js version
+Before upgrading the Node.js version, it is important to decide which version to upgrade to. 
+Broadly speaking, migrate only to even-numbered Node.js releases as only even-numbered versions will be provided with Long-Term Support. 
+
+Read more about Node.js release lines [here](https://nodesource.com/blog/understanding-how-node-js-release-lines-work/).
+
+npm version will be upgraded accordingly to the Node.js version. Hence, do check that this upgrade does not cause any issues.
```

Comment:
Just checking, what does "accordingly to" refer to? Is it automatically upgraded or should I take any action?

```suggestion
The npm version will be upgraded accordingly to the Node.js version. Hence, do check that this upgrade does not cause any issues.
```

Reply:
It should be upgraded automatically! I added further clarification in 
7eff420